### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by mwismann**
# Vet clinic database: query multiple tables
**The following changes were implemented:**

### On `schema.sql`:

- Created the `owners` table ✅ 
- Created the `species` table ✅ 
- Modified the `animals` table: Set `id` as auto-incremented `primary key` ✅ 
- Modified the `animals` table: Removed `species` column ✅ 
- Modified the `animals` table: Added `species_id` column as a `FK` referencing `species` table ✅ 
- Modified the `animals` table: Added `owner_id` column as a `FK` referencing `owners` table ✅ 

### On `data.sql`:

- Inserted data into the `owners` table ✅ 
- Inserted data into the `species` table ✅ 
- Modified the `animals` table: Included `species_id` value ✅ 
- Modified the `animals` table: Incled `owner_id` value ✅ 

### On `queries.sql`:

- Answered the following questions:

1. What animals belong to Melody Pond?
q1

2. List of all animals that are pokemon (their type is Pokemon).
q2

3. List all owners and their animals, remember to include those that don't own any animal.
q3

4. How many animals are there per species?
q4

5. List all Digimon owned by Jennifer Orwell.
q5

6. List all animals owned by Dean Winchester that haven't tried to escape.
q6

7. Who owns the most animals?
q7

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -95,4 +97,50 @@ SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
...
+ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Dean Winchester'
+AND animals.escape_attempts = 0;
+
+-- Who owns the most animals?
+SELECT owners.full_name, COUNT(animals.name)
+FROM owners LEFT JOIN animals
+ON owners.id = animals.owner_id
+GROUP BY owners.full_name
+ORDER BY count DESC;
```
Comment (AbayomiOlaoye) [inline queries.sql]: - [x] You have solid queries here :+1: but thinking about the question, don't you think "Most" ownership should return only one owner not he 6 that you have popping at the moment?
image

I do think you need to check this again. Here's a clue:

```
SELECT owned.full_name
FROM owners AS owned
JOIN animals AS anime on owned.id = anime.owner_id
GROUP BY owned.full_name
ORDER BY COUNT(*) DESC
LIMIT 1;
```
- In fact, what the present query is missing might be the `LIMIT` set to 1. Don't forget to included that query, please :pleading_face:

**Review by AbayomiOlaoye — CHANGES_REQUESTED** [review]
## STATUS: CHANGE REQUIRED ♻️ 

Hi @mwismann :wave: , 

Good job with this milestone so far! What a brilliant attempt!!
There are just a few more issues to work on before moving on to the next project but you are almost there!

so_close

## Highlights
- Descriptive Pull Request message :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- Meaningful commit messages :heavy_check_mark: 
- Good job with the screenshots and clean queries :heavy_check_mark: 
- No linter error :heavy_check_mark: 
- Descriptive ReadMe documentation :heavy_check_mark: 

## Required Changes ♻️
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (mwismann) [reply queries.sql]: Hello @AbayomiOlaoye, thank you for the feedback!

**Review by KDlamini — APPROVED** [review]
### Hi @mwismann 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes for querying multiple tables for the Vet clinic database. 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!🎊 🥳 

## Highlights 🌞 

✔️  Professional pull request
✔️  Correct use of Gitflow 
✔️  All required SQL commands are in their respective files

# Status: Approved ✔️ 

giphyapproved
 
Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**